Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -383,6 +388,9 @@ class WebViewImpl final : public CanMakeWeakPtr<WebViewImpl>, public CanMakeChec
     bool validateUserInterfaceItem(id <NSValidatedUserInterfaceItem>);
     void setEditableElementIsFocused(bool);
 
+    enum class ContentRelativeChildViewsSuppressionType : uint8_t { Remove, Restore, TemporarilyRemove };
+    void suppressContentRelativeChildViews(ContentRelativeChildViewsSuppressionType);
```

Comment:
can this use a template parameter type instead of a normal parameter (since the callsite parameter is never variable)

Reply:
sure

Reply:
Chatted with Richard offline – this offers minor benefit, for more typing. Keeping as-is.